install-deps.sh

```diff
@@ -20,6 +20,8 @@ if test $(id -u) != 0 ; then
     SUDO=sudo
 fi
 export LC_ALL=C # the following is vulnerable to i18n
+#add this to ceph-build or remove all temporary mangling
+export WITH_JAEGER=true
```

Comment:
please split this commit into smaller ones.

Reply:
just meant for testing, removed this shall be handled by ceph-build: https://github.com/ceph/ceph-build/pull/1862

Reply:
this commit is composed multiple changes

1. enable `WITH_JAEGER` 
2. and pass `with_jaeger` down to `munge_ceph_spec_in` for no obvious reasons in `install-deps.sh`
3. mutate `debian/libjaeger.install` in CMakeLists.txt

which of them is for testing ?

Reply:
got your point, enable WITH_JAEGER as we have JAEGER off by default, will split